odoo/addons/base/models/ir_mail_server.py

```diff
@@ -115,6 +116,14 @@ class IrMailServer(models.Model):
     smtp_debug = fields.Boolean(string='Debugging', help="If enabled, the full output of SMTP sessions will "
                                                          "be written to the server log at DEBUG level "
                                                          "(this is very verbose and may include confidential info!)")
+    max_email_size = fields.Float(string="Max email size (MB)", default=5, store=True,
```

Comment:
it's stored by default
```suggestion
    max_email_size = fields.Float(string="Max email size (MB)", default=5,
```

Reply:
Indeed, thanks